lamp/delete-course.php

```diff
@@ -0,0 +1,72 @@
+<?php
+$page_title = "Delete Group";
+$center_page = true;
+?>
+<?php require_once("lib/page-setup.php"); ?>
+<?php include('lib/channel-handler.php'); ?>
+<?php include('lib/course-handler.php');  ?>
+
+<?php
+if (!isset($_GET['course_id'])) {
+    header("Location: courses.php");
+    die();
+}
+$course_id = $_GET["course_id"];
+$course = new Course($course_id);
+$is_instructor = is_user_instructor($course_id);
+include('./common/header.php');
+function renderDeleteCoursePage($error=null) { ?>
+    <h1>Delete a Course</h1>  
+    <?php
+    global $course_id, $course;
+    if ($error) { ?>
+    <br />
+    <div class="alert alert-danger" style="margin-right: 15px;"><?php echo $error; ?></div>
+    <?php } ?>
+    <br />
+    <div class="alert alert-warning" style="margin-right: 15px;">Type 'confirm' into the box below to delete class <?= $course->course_code . "-" . $course->section_number?></div>
```

Comment:
Need to encode html for potentially user-provided content (course code and section number)